arcade/sprite.py

```diff
@@ -1511,8 +1507,4 @@ def get_distance_between_sprites(sprite1: Sprite, sprite2: Sprite) -> float:
     :return: Distance
     :rtype: float
     """
-    distance = math.sqrt(
-        (sprite1.center_x - sprite2.center_x) ** 2
-        + (sprite1.center_y - sprite2.center_y) ** 2
-    )
-    return distance
+    return math.sqrt((sprite1.center_x - sprite2.center_x) ** 2 + (sprite1.center_y - sprite2.center_y) ** 2)
```

Comment:
Original is easier to read. Cases like this, I'm not fond of auto-formatters.

Reply:
I'll revert this thanks.